src/desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -23,25 +25,30 @@ module.exports = class BidForm extends ErrorHandlingForm
   events:
     'click .registration-form-content .avant-garde-button-black': 'placeBid'
     'click .bidding-question': 'showBiddingDialog'
+    'change .registration-form-section__checkbox': 'validateAcceptConditions'
 
-  initialize: ({ @saleArtwork, @bidderPositions, @submitImmediately }) ->
+  initialize: ({ @saleArtwork, @bidderPositions, @submitImmediately, @isRegistered }) ->
     @user = CurrentUser.orNull()
     @$submit = @$('.registration-form-content .avant-garde-button-black')
     @placeBid() if @submitImmediately
     @$('.max-bid').focus() unless @submitImmediately
+    @$acceptConditions = @$('#accept_conditions')
+    @$checkbox = @$('.artsy-checkbox')
```

Comment:
I'm wondering if these names should be more specific or match the mixin name, eg `$conditionsOfSale` and `$conditionsOfSaleCheckbox`

Reply:
Updated to be `conditionsCheckbox`